docs/uds-package-practices.md

```diff
@@ -0,0 +1,60 @@
+# UDS Package Practices
+
+This document describes the practices that a UDS package **must**, **should**, and **may** follow to be considered officially supported as a UDS package and to be able to display the "Made for UDS" badge on its GitHub repository README.md file:
+
+[<img alt="Made for UDS" src="./made-for-uds.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core)
+
+> [!NOTE]
+> This badge should link to the `uds-core` repo and should match the height of any other badges (i.e. `20px`).
+
+## Integrations
+
+Below are the minimal services that a UDS package will integrate with at some level and they:
+
+### Istio
+
+- **Must** define external interfaces under the `expose` key **_(if applicable - i.e. has a web interface)_**.
```

Comment:
may, since _if applicable_ ?

Reply:
Was trying to make it less ambiguous ( didn't want people making their own VSs for example) - we could preface it with the `if`?

Reply:
Actually just reworded them to be a bit clearer